lib/irb/ruby-lex.rb

```diff
@@ -370,7 +370,7 @@ def calc_nesting_depth(opens)
             indent_level = 0
           end
         end
-      when :on_tstring_beg, :on_regexp_beg, :on_symbeg
+      when :on_tstring_beg, :on_regexp_beg, :on_symbeg, :on_backtick
```

Comment:
I forgot to add `:on_backtick` and the test `test_pasted_code_keep_base_indent_spaces` failed
```ruby
          ]+[["a
b" + `c
d` + /e ← failed here
f/ + :"g
h".tap do
```